### PR TITLE
Check if the input number is palindrome or not.

### PR DESCRIPTION
**Pull request by UserRounakk**

### CODE REVIEW

File: C-Programs/Palindrome.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    printf("Enter a number ");
+    scanf("%d",&a);
+    b = a;
+    while (b > 0) {
+    c = b % 10;
+    d = d * 10 + c;
+    b = b/10;  
+    } ;
+    if ( a == d){
+        printf("%d is palindrome",d);
```
Comment (thenishantsapkota) [inline C-Programs/Palindrome.c:14]: Program doesn't work
It shows that every number is palindrome.
Fix that and the PR will be merged!

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Program doesn't work
It shows that every number is palindrome.
Fix that and the PR will be merged!

**Comment by UserRounakk** [comment]
@thenishantsapkota I checked the program and it seems to be working fine. Can you please verify again.

**Comment by thenishantsapkota** [comment]
> @thenishantsapkota I checked the program and it seems to be working fine. Can you please verify again.

image
🤷

**Review by PrajwalCH — COMMENTED** [review]
It always print not palindrome. Please correct your program.

**Comment by UserRounakk** [comment]
@thenishantsapkota Correction done.

**Pull request merged**